### PR TITLE
Updates for Magni robot simulation in Gazebo.

### LINKED ISSUE #69: Gazebo Model for Magni
Being worked on by @Polonium19 

Requires #67

### PR DESCRIPTION
**Pull request by Polonium19**
Magni robot simulation in Gazebo:
- Gazebo package added.
- URDF description update.

Edit (@rohbotics): Closes #69

### CODE REVIEW

**Comment by rohbotics** [comment]
Thanks for the PR Petro.

Trying it out I notice a few problems:
- cmd_vel and odom are namespaced under /ubiquity_velocity_controller when they should be global
- The lighting properties for everything but the casters seems to be wrong, as it is all dark
- There are merge conflicts, so you will have to rebase your commits

Let me know if you need any help fixing those.

Looking forward to seeing sonar support.

File: magni_description/urdf/inertial.xacro [hunk 1]
```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0"?>
+<!--
+Copyright: Osnabrück University, Knowledge Based Systems Group
```
Comment (rohbotics) [inline magni_description/urdf/inertial.xacro]: Add "BSD 3 Clause" to the copyright notice

File: magni_description/urdf/inertial.xacro [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+https://raw.githubusercontent.com/uos/uos_tools/kinetic/uos_common_urdf/common.xacro
+-->
+<robot xmlns:xacro="http://ros.org/wiki/xacro">
```
Comment (rohbotics) [inline magni_description/urdf/inertial.xacro]: This macro has a different scheme url from the other ones. 
```
inconsistent namespace redefinitions for xmlns:xacro:
 old: http://wiki.ros.org/xacro
 new: http://ros.org/wiki/xacro (/home/rohan/melodic_catkin/src/magni_robot/magni_description/urdf/inertial.xacro)
```

File: magni_gazebo/config/magni_controllers.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+ubiquity_joint_publisher:
+  type: "joint_state_controller/JointStateController"
+  publish_rate: 50
+
+ubiquity_velocity_controller:
+  type: "diff_drive_controller/DiffDriveController"
+  left_wheel  : 'left_wheel_joint'
+  right_wheel : 'right_wheel_joint'
+  publish_rate: 50.0
+  pose_covariance_diagonal : [0.001, 0.001, 1000000.0, 1000000.0, 1000000.0, 1000.0]
```
Comment (rohbotics) [inline magni_gazebo/config/magni_controllers.yaml]: Please use the same covariance as the real robot https://github.com/UbiquityRobotics/magni_robot/blob/indigo-devel/magni_bringup/param/base.yaml#L20

**Review by rohbotics — CHANGES_REQUESTED** [review]

**Comment by Polonium19** [comment]
Please check updates for the PR.

**Comment by rohbotics** [comment]
These changes look good thank you. Please add the sensor support.

**Review by rohbotics — APPROVED** [review]

**Pull request merged**